generators/server/templates/src/main/docker/config/realm-config/jhipster-realm.json.ejs

```diff
@@ -1,1873 +1,1861 @@
-            "protocol" : "openid-connect",
-            "protocolMapper" : "oidc-usermodel-property-mapper",
-            "consentRequired" : true,
-            "consentText" : "${username}",
-            "config" : {
-                "userinfo.token.claim" : "true",
-                "user.attribute" : "username",
-                "id.token.claim" : "true",
-                "access.token.claim" : "true",
-                "claim.name" : "preferred_username",
-                "jsonType.label" : "String"
-            }
-        }, {
-            "id" : "019a1374-216c-4478-995e-b0fe5960f4b9",
-            "name" : "full name",
-            "protocol" : "openid-connect",
-            "protocolMapper" : "oidc-full-name-mapper",
-            "consentRequired" : true,
-            "consentText" : "${fullName}",
-            "config" : {
-                "id.token.claim" : "true",
-                "access.token.claim" : "true",
-                "userinfo.token.claim" : "true"
-            }
-        }, {
-            "id" : "1f54330f-3ccc-4b85-a93d-8c20f93abe04",
-            "name" : "family name",
-            "protocol" : "openid-connect",
-            "protocolMapper" : "oidc-usermodel-property-mapper",
-            "consentRequired" : true,
-            "consentText" : "${familyName}",
-            "config" : {
-                "userinfo.token.claim" : "true",
-                "user.attribute" : "lastName",
-                "id.token.claim" : "true",
-                "access.token.claim" : "true",
-                "claim.name" : "family_name",
-                "jsonType.label" : "String"
-            }
-        } ],
-        "useTemplateConfig" : false,
-        "useTemplateScope" : false,
-        "useTemplateMappers" : false
-    }, {
-        "id" : "9080f291-17ef-4022-8afe-7822595006c6",
-        "clientId" : "web_app",
-        "rootUrl" : "http://localhost:<%= serverPort %>",
-        "surrogateAuthRequired" : false,
-        "enabled" : true,
-        "clientAuthenticatorType" : "client-secret",
-        "secret" : "web_app",
-        "redirectUris" : [ "http://localhost:<%= serverPort %>/*", "http://localhost:9000/*", "http://localhost:8100/*", "http://127.0.0.1:8761/*" ],
-        "webOrigins" : [ "http://localhost:<%= serverPort %>/*", "http://localhost:9000/*", "http://localhost:8100/*", "http://127.0.0.1:8761/*" ],
-        "notBefore" : 0,
-        "bearerOnly" : false,
-        "consentRequired" : false,
-        "standardFlowEnabled" : true,
-        "implicitFlowEnabled" : true,
-        "directAccessGrantsEnabled" : true,
-        "serviceAccountsEnabled" : false,
-        "publicClient" : true,
-        "frontchannelLogout" : false,
-        "protocol" : "openid-connect",
-        "attributes" : {
-            "saml.assertion.signature" : "false",
-            "saml.force.post.binding" : "false",
-            "saml.multivalued.roles" : "false",
-            "saml.encrypt" : "false",
-            "saml_for
```

Comment:
This is a regression. Implicit flow is necessary for the Ionic and Ignite JHipster modules. I'll open an issue.

Reply:
I think it's OK to change this in major version, but not in a minor release. I think we should change the behavior back to what it was, then propose the change for 6.0.